lib/ansible/module_utils/redfish_utils.py

```diff
@@ -925,3 +1020,12 @@ def get_system_inventory(self):
 
         result["entries"] = inventory
         return result
+
+    def get_multi_system_inventory(self):
+        ret = True
+        entries = []
+        for systems_uri in self.systems_uris:
+            inventory = self.get_system_inventory(systems_uri)
+            ret = inventory.pop('ret') and ret
+            entries.append(inventory)
```

Comment:
The way the multi-inventory methods are working, the resulting payload looks like this:

```
{
    "ansible_facts": {
        "redfish_facts": {
            "system": {
                "entries": [
                    {
                        "entries": {
                            "AssetTag": "free form asset tag",
                            "BiosVersion": "P79 v1.00 (09/20/2013)",
                            ...
                    },
                    {
                        "entries": {
                            "AssetTag": "free form asset tag",
                            ...
```
 
Note that there 2 levels of `"entries"`. This isn't necessarily bad but could be confusing.
If you changed line 1030 above to this:

```
            entries.append(inventory['entries'])
```

You would get output like this:

```
{
    "ansible_facts": {
        "redfish_facts": {
            "system": {
                "entries": [
                    {
                        "AssetTag": "free form asset tag",
                        "BiosVersion": "P79 v1.00 (09/20/2013)",
                        ...
                    },
                    {
                        "AssetTag": "free form asset tag",
                        ...
```

And I think that would be simpler and more straightforward.

Reply:
Done.